manifest.json

```diff
@@ -437,26 +423,12 @@
         "targetDirPrefix": "ckeditor-sharedspace-"
       },
       "ckeditorBootstrapCK": {
-        "version": "1.0",
+        "version": "1.0.0",
```

Comment:
Don't you need to change the place which imports this too?

(When testing dependencies, always make sure to delete the original folder and redownload from scratch. Otherwise things will "just happen to work" on your computer but won't work on others'.)

Reply:
It's working fine. I tested it by deleting the folder.

Reply:
I tested it by deleting the folder. It works fine (and the version is 1.0.0).

Reply:
I don't understand. So if you rename this version to some arbitrary string (like 1.5.32), it still works fine too?

If not, then why is "1.0.0" counted the same as "1.0"?

Reply:
Yup we can rename to anything it will work fine. The deeper the better so 1.0.0 is better I guess.

Reply:
You know, that doesn't make sense at all to me. How would ckEditor know where to find the files?

Reply:
Ah I see we need to change the path in CKeditor to fetch this.

Reply:
I think it's fixed now. I changed to 1.0.0 at both the locations.

Reply:
OK, I'm still confused. Why did you say it worked fine before? What did you do to verify that, and why did you change your mind about it working fine?

(I ask because if it doesn't actually matter either way what path you use in CKEditor, then maybe we don't need this library at all.)

Reply:
It was my bad. I thought you were saying that file won't download(which even I was surprised). So I ran the server by deleting this folder. The server ran fine downloading the bootstrap, so I said its okay.
Though I'm sure we need this library.
Sorry for the inconvenience caused.

Reply:
OK, thanks for the explanation. Did you manually verify by setting up a local dev server and testing?

If not, please make sure to do that _always_. Otherwise bugs slip through.

Reply:
Yup I did and we need the bootstrap skin.